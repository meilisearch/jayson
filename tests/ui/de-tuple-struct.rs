use jayson::Jayson;

#[derive(Jayson)]
struct TupleStruct(i32, i32);

fn main() {}
