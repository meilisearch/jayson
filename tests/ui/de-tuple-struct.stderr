error: Tuple structs aren't supported by the DeserializeFromValue derive macro
 --> tests/ui/de-tuple-struct.rs:5:19
  |
5 | struct TupleStruct(i32, i32);
  |                   ^^^^^^^^^^
